lua/vyhub/server/sv_api.lua

```diff
@@ -5,6 +5,11 @@ VyHub.API = VyHub.API or {}
 local content_type = "application/json; charset=utf-8"
 local json = VyHub.Lib.json
 
+local httpFunc = HTTP
+if pcall(require, "chttp") and CHTTP ~= nil then
+	httpFunc = CHTTP
+end
+
```

Comment:
This raises an error if the library doesn't exist.
Please check if the binary is installed, e.g. like that:

```
local httpFunc = HTTP
local chttpBin = file.Find("bin/gmsv_chttp_*.dll", "LUA")[1]
if chttpBin ~= nil and pcall(require, "chttp") and CHTTP ~= nil then
	httpFunc = CHTTP
end
```

Reply:
I will do this change myself after merging.